wiki/communities/regional-hub.md

```diff
@@ -109,11 +109,11 @@ German Beat Saber Community, JOIN US NOW! We have Magician! 😎
 
 ### Italy
 
-#### Italian Beat Saber Community
+#### BSITA
 
-A chill and accessible place for Italian players to hang around and meet each other
+A community for Italian Beat Saber players (Gio is here)
```

Comment:
Would rather have it say "The official server of Beat Saber Italia! Open for international players too!" to match the BSMG 
Discord

Reply:
yeah sure